package.json

```diff
@@ -17,20 +17,25 @@
     "@jup-ag/react-hook": "6.0.0-beta.9",
     "@jup-ag/wallet-adapter": "0.2.0",
     "@mercurial-finance/optimist": "0.3.2",
+    "@popperjs/core": "^2.11.8",
     "@project-serum/anchor": "^0.26.0",
+    "@radix-ui/react-popover": "^1.1.1",
     "@solana/spl-token": "0.1.8",
     "@solana/spl-token-registry": "~0.2.4574",
     "@solana/wallet-adapter-wallets": "0.19.22",
     "@solana/web3.js": "1.87.6",
     "@tanstack/react-query": "^4.36.1",
     "bn.js": "5.2.1",
     "bs58": "5.0.0",
+    "class-variance-authority": "^0.7.0",
```

Comment:
dont think these are used, please remove
- class-variance-authority
- lucide-react
- tailwindcss-animate

Reply:
removed the dependencies